src/server/types.ts

```diff
@@ -6,12 +6,6 @@ import {
     WatchOptions,
 } from "./_namespaces/ts";
 
-export interface CompressedData {

```

Comment:
This is used by nobody: https://github.com/search?q=%2F%5Cbts%5C.server%5C.CompressedData%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code